view/frontend/templates/page/analytics.phtml

```diff
@@ -2,30 +2,11 @@
 /**
  * @see Metrilo\Analytics\Block\Analytics
  */
-
-/** @var Metrilo\Analytics\Block\Analytics $events */
-$events = $block->getEvents();
+$event = $block->getEvent();
+$url   = $block->getApiEndpoint() . '/tracking.js?token=' . $block->getApiToken();
```

Comment:
Looking at this, I think it makes sense that the whole URL is returned by either the block. What do you think?